cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/PlatformParametersConsts.java

```diff
@@ -32,6 +32,8 @@
 
     public static final String RESOURCE_CRN_PARAMETER = "resourceCrn";
 
+    public static final String DISK_ENCRYPTION_SET_ID = "diskEncryptionSetId";
+
```

Comment:
I do not see the purpose of this. Please use the constants in `AzureInstanceTemplate`.